0000-template.md

```diff
@@ -3,6 +3,7 @@
 - Name: (fill in the feature name: My Feature)
 - Start Date: (fill in today's date: YYYY-MM-DD)
 - Author(s): (Github usernames)
+- Status: Approved | On Hold
```

Comment:
Should we just have one of these in the template?

It might be confusing for folks to see multiple ones here and as we add more `Status` fields it might get hairy?

Reply:
Where do we show the full list of acceptable values?